docs/source/contribution.rst

```diff
@@ -142,6 +142,27 @@ Please note the followings when writing the document.
   the document.
 
 
+Building Guidelines
+-------------------
+
+In order to run unit tests at the repository root, you first have to build Cython files in place by running the following command::
+
+  $ python setup.py develop
+
+Please note that when you modify ``*.pxd`` files, you must clean ``*.cpp`` and ``*.so`` files once with::
+  
+  $ git clean -fdx
+
+before running ``python setup.py develop`` because currently, we have problems that Cython does not automatically rebuild modified pxd files well.
+
+We do not officially support, but some of the core developer members use `ccache <https://ccache.samba.org/>`_ to boost compilation time.
```

Comment:
I think this part could be made into a `.. note::` section as well.

For English expression, how about:

>It's not officially supported, but you can use `ccache ...` to reduce compilation time.
>On Ubuntu 16.04, you can set up as follows::
>...
>See ... for details.

(It's better to include Ubuntu version. Please confirm the number.)